### DISCUSSION TITLE
Charts using SUM (expenses - income)

### DISCUSSION THREAD

**Discussion by L-E-0-N** [firefly-iii/firefly-iii#8074] [Firefly III]
Hi James, first off loving the app so far!

But I have a quick question:

Is there any way to get the SUM of the categories into the charts? Currently we get 2 pie charts 1 for income and 1 for outcome but ideally I would like to see the SUM (aka net expenses) of the category.

For example for category shopping. I've order clothes for €200,- but also returned some and got back €50,-. One expense of €200,- and one income of €50,-. But still the graphs show €200,- for the 'Shopping category and ideally I would like to see €150,- since we've returned €50,- of clothes.

I've also tried connecting the income to the expense but that didn't work either.

**Comment by JC5** [comment]
Which "the charts" are you referring to specifically?

**Reply by L-E-0-N** [reply]
<img width="1107" alt="Screenshot 2023-10-20 at 10 10 28" src="https://github.com/firefly-iii/firefly-iii/assets/23658130/9969aa91-74c4-4242-b784-b6ad5c228a05">

On the dashboard mainly, the category barchart only shows the expenses which gives an inaccurate representation. 

And within the category report, we get a piechart for expenses and income, but not SUM. But there is a table which shows both so that's less of a worry.

**Reply by JC5** [reply]
Ah yes, I believe I turned that off at some point, but I'll add it again in the new layout. Most charts can do with a tune-up.

**Reply by aronchi-inps** [reply]
Is it now possible to add incomes in category report? it's a very useful feature. I've bought and sold come items in thrift store, but In that report I'm seeing only my expenses.
Thanks in advance!
